_resources/_styles/landingpage.styl

```diff
@@ -0,0 +1,132 @@
+.FlagImgContainer {
+  background: url("/_img/people_only.png");
+  background-repeat: no-repeat;
+  background-size:100%;
+  height: 35vh;
+  width: 100%;
+}
+  @media laptop {
+    .FlagImgContainer {
+      background: url("/_img/people_flag.png") no-repeat;
+      background-position: center center;
+      background-size:100%;
+      height: 100vh;
+      width: 50%;
+      float: left;
+    }
+  }
+.RedBackgroundContainer {
+  background: #FF3700;
+  height: 100vh;
+}
+  @media laptop {
+    .Container {
+      height:100vh;
+      width: 50%;
+      position: relative;
+      float: right;
+    }
+  }
+.Display3 {
+  margin: 7rem 4rem 0 4rem;
+  font-size: 4rem;
+  font-family: serif;
+  width: 75%;
+}
+@media laptop {
+  .Display3 {
+    font-size: 6rem;
+  }
+}
+.TinyLogo{
+  background-image: url("/_img/logo.svg");
+  background-repeat: no-repeat;
+  background-size: 100%;
+  height:18px;
+  width: 26px;
+}
+.TinyLogoLandingPage {
+  position: relative;
+  top: 4rem;
+  left: 4rem;
+}
+label {
+  display: block;
+  margin: 4rem 4rem 0 4rem;
+  font-size: 1.5rem;
+}
+@media laptop {
+  label {
+    margin: 7rem 4rem 0 4rem;
+  }
+}
+.InputZip {
+  margin: 0 2rem 0 4rem;
+  border: none;
+  background: #FF3700;
+  border-bottom: 1px solid white;
+  height: 3rem;
+  width: 21rem;
+  font-size: 1.5rem;
+  color: white;
+}
+.SubmitZip {
+  background: white;
+  border: none;
+  border-radius: 5px;
+  color: #ff3700;
+  height: 3.5rem;
+  width: 6.5rem;
+  font-size: 1.5rem;
+}
+.WhiteText {
+  color: white;
+}
+.LandingPageQuote {
+  margin: 10rem 4rem 0 4rem;
+}
+  @media laptop {
+    .LandingPageQuote {
+      width: 75%;
+      margin: 8rem 4rem 0 4rem
+    }
+  }
+.LandingPageQuoteSpan {
+  font-style: italic;
+}
+.BuiltBy {
+  margin: 6rem 14rem 0 4rem;
+  width: 50%;
+}
+@media laptop {
+  .BuiltBy {
+    width: 75%;
+  }
+}
+.BlackText {
+  color: #000;
+}
+.BorderBottomBlack {
+  border-bottom: 1px solid #000;
+  padding: 0 0 2px 0;
+}
+.BtnGitHub {
+  height: 18px;
+  width: 18px;
+  text-align: center;
+  background: url(_img/github_logo.png) no-repeat;
```

Comment:
you're using double quotes on background images elsewhere. either will work but be consistent.